Makefile

```diff
@@ -87,7 +87,19 @@ ifeq ($(USE_CUDNN), 1)
       $(error ✗ cuDNN not found. See the Makefile for our currently hard-coded paths / install instructions)
     endif
   else
-    $(info → cuDNN is not supported right now outside of Linux)
+    ifneq ($(OS), Windows_NT)
+      $(info → cuDNN is not supported on MAC OS right now)
+    else
+      $(info ✓ Windows cuDNN found, will run with flash-attention)
+      CUDNN_FRONTEND_PATH ?= ..\..\cudnn-frontend\include #override on command line if different location
```

Comment:
seems inconsistent? it's HOME above but it's ../../ here

Reply:
Yes, it's inconsistent - there's not a pre-defined HOME on windows - they have HOMEDRIVE and HOMEPATH. I can use the HOMEPATH since that's more like HOME on Linux if that's preferable? I'll make this change.

Reply:
ok will wait for this before merging

Reply:
This has been fixed.